Mathlib/MeasureTheory/MeasurableSpace/Defs.lean

```diff
@@ -35,6 +35,8 @@ contains all of them.
 measurable space, σ-algebra, measurable function
 -/
 
+assert_not_exists Covariant MonoidWithZero
```

Comment:
We can almost put `assert_not_exists Monoid` here, except `Mathlib.Tactic.Measurability` needs to specially handle `npowRec`...